### ISSUE TITLE
composer install vs composer update

### ISSUE THREAD

**Issue by FinBoWa** [jeffwhitfield/grunt-shipit-silverstripe#1]
Hi,

Shouldn't the deploy process actually run composer install, not update, on the remote.

The deployment should use the lock file to get the tested version of the files, not the latest releases that are found. 

Refering to this line https://github.com/jeffwhitfield/grunt-shipit-silverstripe/blob/bfc88cda82fa6a10df8bc6a25a445165c3966e92/tasks/silverstripe/composer.js#L16

**Comment by jeffwhitfield** [comment]
Yeah, I figured update was more appropriate at the time. Don't see any reason why it can't be switched to install instead of update. I'll make the change. 

That said, fair warning...I'm not really keeping this repo up to date. Been using other build tools instead of ShipIt. In fact, I was planning on building a standard ShipIt compatible module to replace the grunt version. Kinda held off on that though and don't plan on doing anything further with it.
